rotkehlchen/tests/unit/globaldb/test_globaldb.py

```diff
@@ -946,52 +949,52 @@ def test_general_cache(globaldb):
     ts_test_start = ts_now()
     with globaldb.conn.write_ctx() as write_cursor:
         # write some values
-        globaldb_set_general_cache_values(
+        globaldb_set_cache_values(
             write_cursor=write_cursor,
-            key_parts=[GeneralCacheType.CURVE_LP_TOKENS],
+            key_parts=[GeneralCacheType.CURVE_POOL_TOKENS],
             values=['abc'],
         )
-        globaldb_set_general_cache_values(
+        globaldb_set_cache_values(
```

Comment:
Others also asked about this test, but I don't totally get it. I would have to study the code in more detail to understand. I guess you need to test both types, and type depends on key? Is that it? But there is multiple changes. Since actual values do not matter you just test the cache types and that's it?

Reply:
Exactly. It is just about using the write key types to test the cache type.